boards/arm/nucleo_l552ze_q/doc/nucleol552ze_q.rst

```diff
@@ -270,6 +272,11 @@ You should see the following message on the console:
 
    Hello World! arm
 
+Building a secure/non-secure with Arm |reg| TrustZone |reg|
+-----------------------------------------------------------
+
+The TF-M integration sample :ref:`tfm_ipc` can be run by a Nucleo L552ZE Q, using the ``nucleo_l552ze_q_ns`` target.
+
```

Comment:
Since `postbuild.sh` is now run automatically as part of the standard build process, and this script has some requirements, we should at least point the script out here and list some of the requirements it was tested in.

Perhaps something like the following, just to give people an indicator of where to look:

```suggestion
When building a ``*_ns`` image with TF-M, a ``build/tfm/install/postbuild.sh`` bash script will be run as a post-build step to make some required flash layout changes. Check to ensure that the commands required by this script (``readlink``, etc.) are available on your system.
```

You should also list the platforms that this is known to work on or that have been used during testing and development, such as if Linux was used with the Zephyr SDK and which version, etc.

`readlink` in the script is problematic on OS X, for example, since the standard system `readlink` command doesn't have the `-f` flag and you actually need to use `greadlink`, installed via `brew`, which is why we have this logic here for the LPC55S69: https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/master/trusted-firmware-m/platform/ext/target/nxp/lpcxpresso55s69/scripts/build.sh#L11

Reply:
I have updated the documentation according to your suggested change. I have also opened a task in TF-M (T804).